client/components/TestPlanVersionsPage/index.jsx

```diff
@@ -273,10 +273,48 @@ const TestPlanVersionsPage = () => {
                                                     </PhasePill>
                                                 );
 
+                                                const draftPill = (
+                                                    <PhasePill
+                                                        fullWidth={false}
+                                                    >
+                                                        DRAFT
+                                                    </PhasePill>
+                                                );
+
+                                                if (
+                                                    derivedDeprecatedAtPhase ===
+                                                    'RD'
+                                                ) {
+                                                    return (
+                                                        <>
```

Comment:
With NVDA, the text comes back as `Latest Phase  column 2  Deprecatedbefore Draftreview`. The same doesn't happen with VO though, and I haven't checked JAWS.